docs/guides/deploy-hyperlane-local-agents.mdx

```diff
@@ -0,0 +1,397 @@
+
+```bash
+export CONFIG_FILES=/full/path/to/configs/agent-config-{timestamp}.json
+```
+
+### Configure
+
+There are numerous parameters that validators can be configured with. For this guide, we are concerned with just a handful:
+
+| Parameter                       | Description                                                                                                                    |
+| ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `--db`                          | Path for writing persistent data to disk.                                                                                      |
+| `--relayChains`                 | Comma separates names of the origin chain and destination chains to relay messages between. E.g. `ethereum,polygon,avalanche`. |
+| `--allowLocalCheckpointSyncers` | Allows the relayer to look for validator signatures on the local filesystem.                                                   |
+| `--defaultSigner.key`           | A hexadecimal private key used to sign transactions for all chains.                                                            |
+| `--metrics-port`                | Optional. The port to expose prometheus metrics on, defaults to `9090`.                                                        |
+
+To learn more about all the parameters you can change, read the [agent configuration reference](../operate/config-reference.mdx).
+
+<Tabs groupId="docker">
+<TabItem value="docker" label="Using Docker">
+
+**Mounting directories**
+
+For the relayer, we provide almost the same arguments to Docker as the validator:
+
+1. Set the `$CONFIG_FILES` environment variable to a fixed path within the container.
+1. Mount the agent config file to this fixed path and making it **readonly**.
+1. Mount the persistent data directory at a fixed path within the container.
+1. Mount the validator signatures directory to a fixed path within the container and making it **readonly**.
+
+```sh
+...
+-e CONFIG_FILES=/config/agent-config.json \
+--mount type=bind,source=$CONFIG_FILES,target=/config/agent-config.json,readonly \
+--mount type=bind,source="$(pwd)"/hyperlane_db_relayer,target=/hyperlane_db \
+--mount type=bind,source="$(pwd)"/$VALIDATOR_SIGNATURES_DIR,target=/tmp/validator-signatures,readonly \
+...
+```
+
+Hardcoding these paths deduplucates the configuration, making it easier to pass the right arguments when running the container.
+
+</TabItem>
+<TabItem value="from-source" label="Building from source">
+
+**Clone and setup**
+
+If you haven't already done so when setting up the validator, clone the Hyperlane monorepo and follow the [setup instructions](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/README.md) in the `rust` directory.
```

Comment:
nit: let's keep it simple in case someone jumps directly to this section
```suggestion
If you haven't already done so, clone the Hyperlane monorepo and follow the [setup instructions](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/README.md) in the `rust` directory.
```